Source/WebCore/Modules/mediastream/ImageCapture.cpp

```diff
@@ -81,35 +81,42 @@ void ImageCapture::takePhoto(PhotoSettings&& settings, DOMPromiseDeferred<IDLInt
     });
 }
 
-void ImageCapture::getPhotoCapabilities(PhotoCapabilitiesPromise&& promise)
+void ImageCapture::getPhotoCapabilities(DOMPromiseDeferred<IDLDictionary<PhotoCapabilities>>&& promise)
```

Comment:
why not keep the typedef ? a tad less long.

Reply:
It is shorter, but it is only used in this one method